acvm-repo/acvm/src/pwg/mod.rs

```diff
@@ -408,6 +410,12 @@ impl<'a, F: AcirField, B: BlackBoxFunctionSolver<F>> ACVM<'a, F, B> {
         self.status.clone()
     }
 
+    /// Executes a single opcode using the dedicated solver.
+    ///
+    /// Foreign or Acir Calls are deferred to the caller, which will
```

Comment:
```suggestion
    /// Foreign or ACIR Calls are deferred to the caller, which will
```
to be consistent throughout the comments